gulpfile.js

```diff
@@ -19,7 +19,7 @@ const watchPatterns = playbook.content.sources.filter((source) => !source.url.in
 }, [])
 
 
-function generate (done) {
+async function generate (done) {
```

Comment:
not sure how it's fixing the error as it's using callback here

caller is calling generate with a done callback that is invoked when it's finished.
Using async won't solve issue

async would be useful if 
we had for example the code

```javascript
async function generate(): Promise<void> {
  generator(antoraArgs, process.env)
}
```

and then the caller could do
```javascript
await generate();
```

Reply:
I am not sure, but it fixed those errors. @themr0c Fabrice, please, can you provide the feedback about what you saw? :)

Reply:
@benoitf This is copy-paste programming. I couldn't reproduce the error experienced by @MichalMaler. Search on the error message proposed many solutions, we tried prepending the function with async and it fixed his problem. It has a nice secondary effect: the web server is starting before the generation of the files is complete. Any further improvement on the script much appreciated, we reached our limits as tech writers here.